docs/reference/esql/functions/st_x.asciidoc

```diff
@@ -0,0 +1,33 @@
+[discrete]
+[[esql-st_x]]
+=== `ST_X`
+
+*Syntax*
+
+[.text-center]
+image::esql/functions/signature/st_x.svg[Embedded,opts=inline]
+
+*Parameters*
+
+`point`::
+Expression of type `geo_point` or `cartesian_point`. If `null`, the function returns `null`.
+
+*Description*
+
+Special function extracts the `x` coordinate from the supplied point.
```

Comment:
I think the description should be ok without the "Special function" wording. To date I don't think we have any special functions.

Reply:
Indeed, there is nothing particularly 'special' about these functions!
Fixed in 379d6cdd5577a5def64ae4fece1351bac7e64da3